### ISSUE TITLE
A Link Between Worlds "Invalid Texture Format"

### ISSUE THREAD

**Issue by MasterKingSirPlease** [aboood40091/GTX-Extractor#8]
I attempted to extract my .gtx file that I had previously extracted from a .bflim file. I attempted to use your bflim converter to do this, but I was informed that that too was an invalid file format.

**Comment by aboood40091** [comment]
Could you send both files?

**Comment by MasterKingSirPlease** [comment]
Sorry for the late response.
Github is being obnoxious and not allowing me to upload the bflim directly, so I put it in a ZIP file. I deleted the original gtx file and can't find it in my recycle bin, but I found a separate gtx that had the same problem. Ideally, I'd like to skip gtx conversion and work directly with bflim files, making gtx conversion an unnecessary middle step.
[bflim_gtx.zip](https://github.com/aboood40091/GTX-Extractor/files/10573974/bflim_gtx.zip)
